src/interfaces.js

```diff
@@ -83,9 +83,15 @@ export function mapTagOperations({spec, cb = nullFn, defaultTag = 'default'}) {
       // Id already exists?
       if (typeof tagObj[id] !== 'undefined') {
         // Bump counter ( for this operationId )
-        operationIdCounter[id] = (operationIdCounter[id] || 0) + 1
+        let originalCounterValue = (operationIdCounter[id] || 1)
```

Comment:
You can see a test for this logic here: https://github.com/shockey/swagger-js/blob/bug/1003-operation-ids/test/interfaces.js#L345

Two operations with the id `getOne` would yield a `getOne1` and `getOne2`, but no `getOne`. The test confirms this 😄